src/ObjWriting/Game/IW5/AssetDumpers/AssetDumperLeaderboardDef.cpp

```diff
@@ -0,0 +1,39 @@
+#include "AssetDumperLeaderboardDef.h"
+
+#include "Game/IW5/LeaderboardDef/JsonLeaderboardDefWriter.h"
+
+#include <algorithm>
+#include <format>
+#include <ranges>
+
+using namespace IW5;
+
+std::string AssetDumperLeaderboardDef::GetFileNameForAsset(const std::string& assetName)
+{
+    std::string sanitizedFileName(assetName);
+    if (sanitizedFileName[0] == '*')
+    {
+        std::ranges::replace(sanitizedFileName, '*', '_');
+        const auto parenthesisPos = sanitizedFileName.find('(');
+        if (parenthesisPos != std::string::npos)
+            sanitizedFileName.erase(parenthesisPos);
+        sanitizedFileName = "generated/" + sanitizedFileName;
+    }
+
+    return std::format("leaderboards/{}.json", sanitizedFileName);
+}
```

Comment:
This will not be necessary, it only was for materials because in T6 Radiant generated materials can have `*` and `()` in them.

You can just use `std::format("leaderboards/{}.json")` and inline that where the function was used